### ISSUE TITLE
Missing use statement in EnvironmentDetector.php

### ISSUE THREAD

**Issue by vladigor** [acquia/drupal-recommended-settings#34]
**Describe the bug**
When running a site install I hit this error:
```
Error: Class "Acquia\Drupal\RecommendedSettings\Helpers\ConfigInitializer" not found in Acquia\Drupal\RecommendedSettings\Helpers\EnvironmentDetector::getSiteName() (line 230 of /app/vendor/acquia/drupal-recommended-settings/src/Helpers/EnvironmentDetector.php).
```

I believe the fix is to insert the following at line 6 of  EnvironmentDetector.php
`use Acquia\Drupal\RecommendedSettings\Config\ConfigInitializer;`

**Comment by vishalkhode1** [comment]
Hi @vladigor 
This is now fixed in the `develop` branch. Soon, we'll be making a stable release. Will keep you posted once this is released. Thanks.

**Comment by vishalkhode1** [comment]
This is planned for release this week.

**Comment by vishalkhode1** [comment]
Fixed and released in [1.0.0](https://github.com/acquia/drupal-recommended-settings/releases/tag/1.0.0).
